### ISSUE TITLE
DITA to Markdown breaks tables containing block elements

### ISSUE THREAD

**Issue by raducoravu** [jelovirt/org.lwdita#76]
I have this DITA simpletable:
```xml
<simpletable frame="all" relcolwidth="1* 1* 1*" id="simpletable_wkl_j5w_zmb">
    <sthead>
        <stentry>Version</stentry>
        <stentry>Date</stentry>
        <stentry>Description of Revision</stentry>
    </sthead>
    <strow>
        <stentry>1</stentry>
        <stentry>1</stentry>
        <stentry>
            <ul id="ul_rnh_fvw_zmb">
                <li>item1</li>
                <li>item2</li>
            </ul>
        </stentry>
    </strow>
    <strow>
        <stentry>2</stentry>
        <stentry>2</stentry>
        <stentry>cell content</stentry>
    </strow>
</simpletable>
```
which when published to Markdown produces this content:
```markdown
# Introduction {#introduction}

|Version|Date|Description of Revision|
|-------|----|-----------------------|
|1|1|-   item1
-   item2

|
|2|2|cell content|
```
      
because of those new lines in the table cell, the table is broken. 
It's quite silly but from what I've googled the only way this could work would be if the generated Markdown content looked like this:
```markdown
# Introduction {#introduction}

|Version|Date|Description of Revision|
|-------|----|-----------------------|
|1|1|<ul><li>item1</li><li>item2</li></ul>|
|2|2|cell content|
```
and HTML would be produced for the table cell content if it has new lines in it.

**Comment by raducoravu** [comment]
👍 another one of our end users wanted to publish DITA tables containing complex cells like this:

                                        <entry>
                                            <p>Some<ul
                                                  id="ul_y4x_nrd_q4b">
                                                  <li>list item</li>
                                                  <li>list item</li>
                                                </ul><note type="note">It is recommended....</note></p>
                                        </entry>

to Markdown.

**Comment by jelovirt** [comment]
Implemented in #200
